catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java

```diff
@@ -199,8 +199,19 @@
      */
     public static final String DEFAULT_TAG = "resource";
 
+
+    /**
+     * {@link Attribute} algorithm used to calculate the checksum on the {@link Metacard#RESOURCE_URI}
```

Comment:
I think this is misleading - we will only calculate the checksum on local resources correct?